internal/backends/collection.go

```diff
@@ -64,6 +64,8 @@ type QueryParams struct {
 
 // QueryResult represents the results of Collection.Query method.
 type QueryResult struct {
+	// nothing for now - no pushdowns yet
+	// TODO https://github.com/FerretDB/FerretDB/issues/3235
```

Comment:
What will be added there once we add pushdown?

Reply:
But why would we need that?

Reply:
to show results of push down in explain?

Reply:
`QueryResults` is a result of `Query`, not `Explain`

Reply:
ok, then I remove this todo for now, and we can decide later where these param should belong.